### PR TITLE
Add multiple read from view

### PR DESCRIPTION
**Pull request by mffrench**
Hello,

I added a function to read a view with a POST and an array of keys.

Let me know what you think.

Cheers,

### CODE REVIEW

File: couchdb.go [hunk 1]
```diff
@@ -717,13 +717,49 @@ func (db *Database) GetView(designDoc string, view string,
...
+
 	err = parseBody(resp, &results)
 	if err != nil {
 		return err
 	}
 	return nil
 }
 
+//Get multiple results of a view.
+func (db *Database) GetMutipleFromView(designDoc string, view string,
```
Comment (rhinoman) [inline couchdb.go]: Should be GetMultipleFromView

**Review by rhinoman — CHANGES_REQUESTED** [review]
Looks good, just need to correct the spelling of the function name.  Thanks!

File: couchdb.go (hunk 1 above)
Comment (mffrench) [reply couchdb.go]: yeah you right ! will do that asap :)

**Comment by mffrench** [comment]
Just solved spelling issue...

**Review by rhinoman — APPROVED** [review]
Thanks

**Pull request merged**